module-code/app/securesocial/controllers/Assets.scala

```diff
@@ -1,3 +1,859 @@
-package securesocial.controllers
+/*
+ * Copyright (C) 2009-2017 Lightbend Inc. <https://www.lightbend.com>
+ */
+import java.io._
+import java.net.{ JarURLConnection, URL, URLConnection }
+import java.util.Date
+import java.util.regex.Pattern
+import java.time._
+import java.time.format.{ DateTimeFormatter, DateTimeParseException }
+import javax.inject.{ Inject, Singleton, Provider }
 
-class Assets extends controllers.AssetsBuilder(play.api.http.LazyHttpErrorHandler)
\ No newline at end of file
+import play.api._
+import play.api.libs._
+import play.api.mvc._
+import play.core.routing.ReverseRouteContext
+import play.utils.{ InvalidUriEncodingException, Resources, UriEncoding }
+
+import scala.collection.concurrent.TrieMap
+import scala.concurrent.{ ExecutionContext, Future, Promise, blocking }
+import scala.util.control.NonFatal
+import scala.util.{ Failure, Success }
+import scala.concurrent.ExecutionContext.Implicits.global
+
+package securesocial.controllers {
+
+
+  import akka.stream.Materializer
+  import akka.stream.scaladsl.StreamConverters
+  import play.api
+  import play.api.http._
+  import play.api.i18n.MessagesApi
+  import play.api.inject.{ApplicationLifecycle, Module}
+  import play.api.libs.Files.TemporaryFileCreator
+  import play.api.ApplicationLoader
+  import play.api.Configuration
+  import play.api.inject._
+
+  class SecureSocialApplicationLoader extends play.api.inject.guice.GuiceApplicationLoader() {
+    override def builder(context: ApplicationLoader.Context): play.api.inject.guice.GuiceApplicationBuilder = {
+      val extra = Configuration("a" -> 1)
```

Comment:
Maybe, it's for debugging?

Reply:
No. This code works. This code specifies Guice to added custom graph descriptions.